contracts/utils/fieldOperations/Precompiled.sol

```diff
@@ -93,6 +93,19 @@ library Precompiled {
         return abi.decode(output, (uint256)) != 0;
     }
 
+    // rngOnChain should be SKALE Random Number Generator predeployed or similar
+    function getRandomBytes(address rngOnChain) internal view returns (bytes32 addr) {
```

Comment:
This library only calls precompiles.
Not sure that putting this logic here is a good idea.